nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -131,8 +131,10 @@ nifi.web.jetty.threads=${nifi.web.jetty.threads}
 
 # security properties #
 nifi.sensitive.props.key=
+nifi.sensitive.props.key.protected=${nifi.sensitive.props.key.protected}
```

Comment:
Thanks for catching this Bryan. It was an issue where the tool had to merge two instances of `Properties` together and the logic wasn't checking the right key set. Added a test and fixed. Empty protection values no longer stop the field from being protected.